graphql/schemabuilder/types.go

```diff
@@ -299,6 +301,18 @@ type method struct {
 	ManualPaginationArgs manualPaginationArgs
 }
 
+type concurrencyArgs struct {
+	concurrencyFunc ConcurrencyController
+}
+
+// ConcurrencyController is a configuration option for non-expensive and batch
```

Comment:
Let's use a more self-describing name, eg. NumGoroutines() or NumParallelInvocations()?

Thinking about this some more, I think the API I would want to use is a MaxBatchSize()? I suspect if we default that to 1,000 it will do the right thing for pretty much most use cases.

Reply:
Yeah, can change the name, NBD.

For the MaxBatchSize option.  It should be pretty trivial to write a internal function that uses the `ConcurrencyController` under the hood, but, can be embedded directly as `internalpackage.MaxBatchSize`.